doc/Metrics.md

```diff
@@ -0,0 +1,67 @@
+## Metrics in EdgeHub
+
+EdgeHub is intrumented to collect the following metrics:
+
+| MetricName        | Description           | Unit  |
+| ------------- |:-------------:|:-----:|
+| EdgeHubToCloudMessageLatencyMs | Time taken by EdgeHub to send a message to the cloud | Milliseconds |
```

Comment:
Are the Latency measurements average over the reporting period, or are we reporting per message?

Reply:
Latency is collected per message and not averaged over the period.